package.json

```diff
@@ -101,36 +101,36 @@
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.4",
-    "css-loader": "^1.0.0",
-    "eslint": "^5.6.1",
+    "cross-env": "^5.2.0",
+    "css-loader": "^1.0.1",
+    "eslint": "^5.8.0",
     "eslint-config-cheminfo": "^1.18.0",
-    "eslint-config-neptune-react": "^2.4.0",
+    "eslint-config-neptune-react": "^2.5.0",
     "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^21.24.1",
+    "eslint-plugin-jest": "^21.26.2",
     "eslint-plugin-react": "^7.11.1",
     "jest": "^23.6.0",
     "jquery": "^3.3.1",
-    "make-promises-safe": "^1.1.0",
+    "make-promises-safe": "^3.0.0",
     "prettier-eslint-cli": "^4.7.1",
     "prop-types": "^15.6.2",
-    "react": "^16.5.2",
-    "react-dom": "^16.5.2",
-    "react-redux": "^5.0.7",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "react-redux": "^5.1.0",
     "react-router-dom": "^4.3.1",
     "react-select2-wrapper": "^1.0.4-beta6",
-    "redux": "^4.0.0",
-    "redux-actions": "^2.6.1",
+    "redux": "^4.0.1",
+    "redux-actions": "^2.6.3",
     "redux-persist": "^5.10.0",
     "redux-promise-middleware": "^5.1.1",
     "redux-thunk": "^2.3.0",
     "rimraf": "^2.6.2",
-    "should": "^13.2.3",
-    "style-loader": "^0.23.0",
+    "style-loader": "^0.23.1",
     "supertest": "^3.3.0",
     "terser-webpack-plugin": "^1.1.0",
-    "webpack": "^4.20.2",
+    "webpack": "^4.23.1",
     "webpack-cli": "^3.1.2",
-    "webpack-serve": "^2.0.2",
+    "webpack-dev-server": "^3.1.10",
```

Comment:
Isn't webpack-serve the way to go with webpack 4?

Reply:
webpack-serve is depracated and points to webpack-dev-server, which is documented officially on https://webpack.js.org/configuration/dev-server/